theme/less/_homepage.less

```diff
@@ -0,0 +1,92 @@
+main
+{
+  & > section
+  {
+    padding: 78px 25px;
+    box-sizing: border-box;
+  }
+
+  h2, h3, h4
+  {
+    font-weight: 700;
+  }
+
+  h2
+  {
+    font-size: 3.75rem;
+    line-height: 120%;
+    margin: 0;
+  }
+
+  h3
+  {
+    font-size: 3rem;
+  }
+
+  h3 em
+  {
+    font-style: normal;
+    color: @secondary;
+  }
+
+  h4
+  {
+    font-size: 2.25rem;
+  }
+
+  ul li
+  {
+    font-weight: 300;
+    font-size: 1.5rem;
+    line-height: 184.9%;
+    list-style: none;
+  }
+
+  ul li::before
+  {
+    content: "";
+    background: url("/images/list-point.svg") no-repeat left center transparent;
+    display: inline-block;
+    width: 15px; 
+    height: 15px;   
+  }
+
+  ul
+  {
+    padding: 40px 30px;
+    margin: 0;
+  }
+
+  #promo-video
+  {
+    width: 100%;
+    max-width: 448px;
+    height: 281.6px;
```

Comment:
Detail: Let's not be that precise i.e: `1.6px`  this way it will be harder to spot common sizes and patterns, I'd use `280px` instead, same applies to other similar calculations.

Reply:
Done.